testing/upload_validation.ts

```diff
@@ -817,10 +817,10 @@ const genericObjectSchema: z.ZodTypeAny = z.lazy(() =>
     ...basePropertyValidators,
     type: zodDiscriminant(ValueType.Object),
     description: z.string().optional(),
-    id: z.string().optional(),
-    idProperty: z.string().optional(),
-    primary: z.string().optional(),
-    displayProperty: z.string().optional(),
+    id: z.string().min(1).optional(),
```

Comment:
I think Zod has `.nonempty()` for strings.

Reply:
And this one?

Reply:
`.nonempty` is deprecated